packages/lib-sourcify/src/lib/types.ts

```diff
@@ -546,51 +546,51 @@ export interface StorageLayout {
     };
   };
 }
-interface CompilerOutputContracts {
-  [globalName: string]: {
-    [contractName: string]: {
-      abi: Abi;
-      metadata: string;
-      userdoc?: any;
-      devdoc?: any;
-      ir?: string;
-      irAst?: any;
-      irOptimized?: string;
-      irOptimizedAst?: any;
-      storageLayout?: StorageLayout;
-      evm: {
-        assembly?: string;
-        legacyAssembly?: any;
-        bytecode: CompilerOutputEvmBytecode;
-        deployedBytecode?: CompilerOutputEvmDeployedBytecode;
-        methodIdentifiers?: {
-          [methodName: string]: string;
-        };
-        gasEstimates?: {
-          creation: {
-            codeDepositCost: string;
-            executionCost: string;
-            totalCost: string;
-          };
-          external: {
-            [functionSignature: string]: string;
-          };
-          internal: {
-            [functionSignature: string]: string;
-          };
-        };
+export interface SolidityOutputContracts {
```

Comment:
`SolidityOutputContract` seems more appropriate because it represents only one contract